modules/kubernetes-addons/csi-secrets-store-provider-aws/main.tf

```
@@ -1,26 +1,27 @@
 locals {
-  name      = try(var.helm_config.name, "csi-secrets-store-provider-aws")
-  namespace = try(var.helm_config.namespace, "kube-system")
+  name      = try(var.helm_config.name, "secrets-store-csi-driver-provider-aws")
+  namespace = try(var.helm_config.namespace, local.name)
 }
 
 resource "kubernetes_namespace_v1" "csi_secrets_store_provider_aws" {
+  count = try(var.helm_config["create_namespace"], true) && local.namespace != "kube-system" ? 1 : 0
   metadata {
     name = local.namespace
   }
 }
 
 module "helm_addon" {
-  source = "../helm-addon"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints/modules/kubernetes-addons/helm-addon"
```

You should revert this to be a local path
```suggestion
  source = "../helm-addon"
```